src/test/java/uk/gov/gchq/palisade/policy/WrappedRuleTest.java

```diff
@@ -26,10 +26,17 @@
 
     @Test
     public void shouldConstruct1ArgumentWithNoErrors() throws Exception {
-        new WrappedRule<>(null, null, null);
-        new WrappedRule<>(new TestRule(), null, null);
-        new WrappedRule<>(null, o -> o.toString(), null);
-        new WrappedRule<>(null, null, o -> true);
+        // Given
+        WrappedRule rule1 = new WrappedRule<>(null, null, null);
+        WrappedRule rule2 = new WrappedRule<>(new TestRule(), null, null);
+        WrappedRule rule3 = new WrappedRule<>(null, o -> o.toString(), null);
+        WrappedRule rule4 = new WrappedRule<>(null, null, o -> true);
+
+        // Then
+        Assert.assertNotNull(rule1);
+        Assert.assertNotNull(rule2);
+        Assert.assertNotNull(rule3);
+        Assert.assertNotNull(rule4);
```

Comment:
we should make these assertions check that the wrappedRule has set the correct field correctly.